### DISCUSSION TITLE
Start with specific index preselected

### DISCUSSION THREAD

**Discussion by bschlenk** [junegunn/fzf#3272] [Q&A] [answered]
Is there a way to start fzf with a specific index preselected? I'm thinking of adding a few separate actions to the top of my fzf list, and would like to start with the first non-action item selected. I didn't see anything like this in the man pages (searched for both "start" and "index").

**Comment by LangLangBart** [comment, marked as answer]
- `pos(...)` ?
  - added with version [0.36.0](https://github.com/junegunn/fzf/releases/tag/0.36.0)

```sh
seq 10 | fzf --bind 'load:pos(3)'
```
